shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -145,48 +146,30 @@ - (UIAccessibilityTraits)accessibilityTraits {
 @end  // FlutterSwitchSemanticsObject
 
 @interface FlutterScrollableSemanticsObject ()
-@property(nonatomic, strong) SemanticsObject* semanticsObject;
+@property(nonatomic, strong) FlutterSemanticsScrollView* scrollView;
 @end
 
 @implementation FlutterScrollableSemanticsObject {
   fml::scoped_nsobject<SemanticsObjectContainer> _container;
 }
 
-- (instancetype)initWithSemanticsObject:(SemanticsObject*)semanticsObject {
-  self = [super initWithFrame:CGRectZero];
+- (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)bridge
+                           uid:(int32_t)uid {
+  self = [super initWithBridge:bridge uid:uid];
   if (self) {
-    _semanticsObject = [semanticsObject retain];
-    [semanticsObject.bridge->view() addSubview:self];
-    [self setShowsHorizontalScrollIndicator:NO];
-    [self setShowsVerticalScrollIndicator:NO];
+    _scrollView = [[FlutterSemanticsScrollView alloc] initWithSemanticsObject:self];
+    [_scrollView setShowsHorizontalScrollIndicator:NO];
+    [_scrollView setShowsVerticalScrollIndicator:NO];
+    [self.bridge->view() addSubview:_scrollView];
   }
   return self;
 }
 
 - (void)dealloc {
-  _container.get().semanticsObject = nil;
-  [_semanticsObject release];
-  [self removeFromSuperview];
+  [_scrollView removeFromSuperview];
```

Comment:
you need a `[_scrollView release]`, you are leaking it currently

Reply:
oops nice catch